editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -839,6 +840,7 @@ const GridControl = React.memo<GridControlProps>(({ grid }) => {
               style={{
                 position: 'relative',
                 pointerEvents: 'initial',
+                zIndex: activePositioningTarget ? 1 : 0,
```

Comment:
would it be better to leave it as `undefined` or `inherit` in the right hand case?

Reply:
hm yeah that might be better